kount/util/xmlparser.py

```diff
@@ -1,14 +1,13 @@
 ﻿#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """parse Kount's validation xml to python dict and use
-this dict in the package to avoid parsing. New XML_DICT must be generated
-if the xml file 'validate.xml' is changed."""
+this dict in the package to avoid parsing. New XML_DICT will be
+generated automatically if the xml file 'validate.xml' is changed."""
```

Comment:
This is not the exact behavior that we discussed the other day. We don't need automatic regeneration of the dictionary if the validate.xml has changed during the application execution. See https://github.com/Kount/kount-ris-python-sdk/pull/3/files#r121589832 for details on the flow.